lib/susedistribution.pm

```diff
@@ -576,38 +554,44 @@ sub init_consoles {
             });
     }
 
-    if (get_var('BACKEND', '') =~ /ipmi|s390x|spvm|pvm_hmc/ || get_var('S390_ZKVM')) {
+    if (get_var('BACKEND', '') =~ /ipmi|s390x|spvm|pvm_hmc/ || get_var('SUT_IP') || get_var('VIRSH_GUEST')) {
         my $hostname;
-
-        $hostname = get_var('VIRSH_GUEST') if get_var('S390_ZKVM');
+        $hostname = get_var('SUT_IP', get_var('VIRSH_GUEST')) if get_var('SUT_IP') || get_var('VIRSH_GUEST');
         $hostname = get_required_var('SUT_IP') if get_var('BACKEND', '') =~ /ipmi|spvm|pvm_hmc/;
 
         if (is_backend_s390x) {
-
             # expand the S390 params
             my $s390_params = get_var("S390_NETWORK_PARAMS");
             my $s390_host = get_required_var('S390_HOST');
             $s390_params =~ s,\@S390_HOST\@,$s390_host,g;
             set_var("S390_NETWORK_PARAMS", $s390_params);
 
             ($hostname) = $s390_params =~ /Hostname=(\S+)/;
-
-            # adds serial console for S390_ZKVM
-            # NOTE: adding consoles just at the top of init_consoles() is not enough, otherwise
-            # using just them would fail with:
-            # ::: basetest::runtest: # Test died: Error connecting to <root@192.168.112.9>: Connection refused at /usr/lib/os-autoinst/testapi.pm line 1700.
-            unless (get_var('SUT_IP')) {
-                $self->add_console(
-                    'root-serial-ssh',
-                    'ssh-serial',
-                    {
-                        hostname => $hostname,
-                        password => $testapi::password,
-                        username => 'root'
-                    });
-            }
```

Comment:
damn, I am wondering what else might break over here. I guess we can just try it out and be ready to revert if anything

Reply:
hm...maybe there should be more `svirt` VRs ?

Reply:
I tried o cover it will test on "each" backend, but ofc it can still break somewhere. :crossed_fingers:

Reply:
Think I tested each svirt, but I can check again.

Reply:
I went trough the worker configuration https://gitlab.suse.de/openqa/salt-pillars-openqa/-/blob/master/openqa/workerconf.sls
Added few tests, I could not clone some tests because e.g. ipmi workers are down I guess due to migration.
But there is one ipmi tests and I assume they should behave same.
svirt-hyperv tests are failing but this test are AFAIS very bad, e.g. bootloader is finished but in following test is still typing in grub2, behaves same wihtout my change. :face_with_spiral_eyes: 
I would merge this next week :game_die:  :pray:

Reply:
There is no `svirt-hyperv2012r2`, that is special, I found only old 15-SP5 tests on build 102.1